src/frontend/gatsby-config.js

```diff
@@ -1,5 +1,11 @@
 const path = require('path');
 
+const Version = require('../../package.json');
+
+const getRepoInfo = require('../../node_modules/git-repo-info');
```

Comment:
You should be able to `require('git-repo-info')` and the way that node resolves is that it will go up to parent directories to find it.  That said, why not just install this in the frontend package.json vs. the root?